### PR TITLE
[DSD-3920] Added script for hashid

### PR DESCRIPTION
**Pull request by abhishek8shankar**

### CODE REVIEW

File: charts/dslrig/templates/rolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Release.Name }}
+    namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: Role
+  name: {{ .Release.Name }}-secrets-pods-role
+  apiGroup: rbac.authorization.k8s.io
+
+---
```
Comment (syedsalman3753) [inline charts/dslrig/templates/rolebinding.yaml:29]: please remove this

File: charts/dslrig/templates/role.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+kind: Role
+metadata:
+  name: {{ .Release.Name }}-secrets-pods-role
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups: [""]
+    resources: ["secrets","pods"]
+    verbs: ["get","list","watch"]
+
+---
```
Comment (syedsalman3753) [inline charts/dslrig/templates/role.yaml:23]: please remove this

File: charts/apitestrig/templates/rolebinding.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Release.Name }}
+    namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: Role
+  name: {{ .Release.Name }}-secrets-pods-role
+  apiGroup: rbac.authorization.k8s.io
+
+---
```
Comment (syedsalman3753) [inline charts/apitestrig/templates/rolebinding.yaml:29]: please remove this

File: charts/apitestrig/templates/role.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+kind: Role
+metadata:
+  name: {{ .Release.Name }}-secrets-pods-role
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups: [""]
+    resources: ["secrets","pods"]
+    verbs: ["get","list","watch"]
+
+---
```
Comment (syedsalman3753) [inline charts/apitestrig/templates/role.yaml:23]: please remove this

File: charts/apitestrig/templates/role.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Release.Name }}-deployment-role
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups: ["apps"]
+    resources: ["deployments"]
```
Comment (syedsalman3753) [inline charts/apitestrig/templates/role.yaml:8]: resources: ["deployments", "pods"]

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: charts/dslrig/templates/role.yaml (hunk 2 above)
Comment (abhishek8shankar) [reply charts/dslrig/templates/role.yaml]: removed

File: charts/apitestrig/templates/rolebinding.yaml (hunk 3 above)
Comment (abhishek8shankar) [reply charts/apitestrig/templates/rolebinding.yaml]: removed

File: charts/apitestrig/templates/role.yaml (hunk 4 above)
Comment (abhishek8shankar) [reply charts/apitestrig/templates/role.yaml]: removed

File: charts/apitestrig/templates/role.yaml (hunk 5 above)
Comment (abhishek8shankar) [reply charts/apitestrig/templates/role.yaml]: added

**Pull request closed without merging**